documents/payment.md

```diff
@@ -863,6 +863,259 @@ Razorpay::Iin.fetch(tokenIin);
 }
 ```
 -------------------------------------------------------------------------------------------------------
+-------------------------------------------------------------------------------------------------------
+
+### Fetch a Payment (With Expanded EMI Details)
+
+```rb
+Razorpay::Payment.expendDetails("pay_XXXXXXXXXXXXXX",{"expand[]": "emi"})
```

Comment:
please update the function name here

Reply:
updated

https://github.com/razorpay/razorpay-ruby/blob/4baba137cfd918b98d601eca7764e86267276e6f/documents/payment.md?plain=1#L871